llm_bench/python/convert.py

```diff
@@ -48,13 +41,57 @@
 from utils.nncf_utils import COMPRESSION_OPTIONS, INT4_MODEL_CONFIGURATION, get_compressed_path
 from utils.conversion_utils.convert_patch import patch_model_for_optimum_export
 from utils.conversion_utils.better_transformer_patch import patch_model_with_bettertransformer
+from utils.conversion_utils.export_configs import *
+from utils.ov_model_classes import register_normalized_configs
+
+register_normalized_configs()
 
 
 class BackendType(Enum):
     PYTORCH = 'pytorch'
     OPENVINO = 'openvino'
 
 
+def is_ov_model_provided(model_id, model_dir, precision, model_name="openvino_model.xml"):
+    model_dirs = []
+    if Path(model_id).is_dir():
+        model_dirs.append(Path(model_id))
+        model_dirs.append(Path(model_id) / precision)
+        model_dirs.append(Path(model_id) / 'dldt' / precision)
```

Comment:
A good practice is to introduce constants for folder names ('dldt', 'pytorch', etc.) and use them across the code. I guess you can do it in a separate PR.